lib/components/term.js

```diff
@@ -200,6 +200,12 @@ export default class Term extends Component {
     if (this.props.customCSS !== nextProps.customCSS) {
       this.term.prefs_.set('user-css', this.getStylesheet(nextProps.customCSS));
     }
+
+    if (this.props.bell === 'SOUND') {
+      this.term.prefs_.set('audible-bell-sound', this.props.bellSoundURL);
+    } else {
+      this.term.prefs_.set('audible-bell-sound', '');
+    }
```

Comment:
Oh, added that https://github.com/zeit/hyperterm/pull/468/files#diff-268710614c2274f611be667a5f72bb13R18